### ISSUE TITLE
The alt text width limit sometimes fails

### ISSUE THREAD

**Issue by href** [OneGov/onegov.town#220]
This seems to happen if the image loads very slowly. In this case the width we set is wrong.

Ideally we would know for sure once the image has been downloaded and only then take the width.
